### ISSUE TITLE
When "ignore emotes" enabled, words "to" and "for" are omitted

### ISSUE THREAD

**Issue by ibillingsley** [mentegago/streamkit#10]
When "Ignore emotes" is checked the TTS doesn't read out the words "to" and "for", even in the middle of other words.
Some example messages:
for to --> (nothing)
comfortable --> com table
tonight --> night
stop --> s p

edit: another weird one I found
this! --> thi

**Comment by ibillingsley** [comment]
I think I figured out the cause. Because I don't have any bttv emotes on my channel, 
`https://decapi.me/bttv/emotes/iza_k` returns 
`Unable to retrieve BetterTTV details for channel: iza_k`
So the words in this response are treated as emotes and filtered out.

**Comment by ibillingsley** [comment]
Finally, the issue with `this!` is because the global bttv emotes include these modifiers:
```json
{"id":"6468f7acaee1f7f47567708e","code":"c!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true},
{"id":"6468f845aee1f7f47567709b","code":"h!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true},
{"id":"6468f869aee1f7f4756770a8","code":"l!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true},
{"id":"6468f883aee1f7f4756770b5","code":"r!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true},
{"id":"6468f89caee1f7f4756770c2","code":"v!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true},
{"id":"6468f8d1aee1f7f4756770cf","code":"z!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true},
{"id":"64e3b31920cb0d25d950a9f9","code":"w!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true},
{"id":"65cbe7dbaed093b2eaf87c65","code":"p!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true},
{"id":"662953eeea369c0ece39eb93","code":"s!","imageType":"png","animated":false,"userId":"5561169bd6b9d206222a8c19","modifier":true}
```
Perhaps the emotes with `"modifier":true` should be excluded.

**Comment by mentegago** [comment]
Thanks for reporting. Let me look into it.

**Comment by ibillingsley** [comment]
So before I ever signed in to betterttv.com, `https://decapi.me/bttv/emotes/iza_k` returned
`Unable to retrieve BetterTTV details for channel: iza_k` (status code 200)

Now that I've signed in but not added any emotes, it returns
`iza_k does not have any BetterTTV emotes.` (status code 200)

Since these all have status code 200, the only way to distinguish from a valid list of emotes might be to check that the response text doesn't start with "Unable to retrieve" or end with "does not have any BetterTTV emotes."

BTW there is a similar API available at `https://twitch.center/customapi/bttvemotes?channel=iza_k` but it also has an ambiguous response when there are no emotes (`user not found` or `No emotes found` with status code 200)


For now I have added some bttv emotes to work around this issue.

**Comment by mentegago** [comment]
@ibillingsley Thanks for your investigation! I've never tested on an account without BTTV emotes before, so I did not expect such edge case.

For now, I'll probably perform static string check and hope decapi never changes its error response.

Though I can probably implement some sort of remote config later on, so I don't need to create a new version release if they ever change the error response.

**Comment by mentegago** [comment]
This is fixed in the upcoming 0.8 release. Thanks a lot for your report 🙇.
